acme/resource_acme_certificate.go

```diff
@@ -278,6 +283,19 @@ func resourceACMECertificateCreate(d *schema.ResourceData, meta interface{}) err
 		opts = append(opts, dns01.AddRecursiveNameservers(s))
 	}
 
+	if preCheckDelay := d.Get("pre_check_delay").(int); preCheckDelay > 0 {
+		delay := time.Duration(preCheckDelay) * time.Second
+		wrapFunc := func(domain, fqdn, value string, orig dns01.PreCheckFunc) (bool, error) {
+			stop, err := orig(fqdn, value)
+			if stop && err == nil {
+				log.Printf("[%s] acme: Waiting an additional %s for DNS record propagation.", domain, delay)
```

Comment:
This log entry should be prefixed with the `[DEBUG]` log level.
```suggestion
				log.Printf("[DEBUG] [%s] acme: Waiting an additional %s for DNS record propagation.", domain, delay)
```

Reply:
@vancluever thanks for the review. I probably won't get to this since we switched to having our customers install certmanager out of band. Should I close?

Reply:
@tjhiggins don't close it out! I'll finish it up and merge it with your commit. Thanks for the work you've done so far!

Reply:
Going to keep this log entry like this particularly as it's run in the lego code path and this is generally how lego structures their log messages.